src/buildercore/cfngen.py

```diff
@@ -366,7 +373,7 @@ class Delta(namedtuple('Delta', ['plus', 'edit', 'minus'])):
     def non_empty(self):
         return self.plus['Resources'] or self.plus['Outputs'] or self.edit['Resources'] or self.edit['Outputs'] or self.minus['Resources'] or self.minus['Outputs']
 
-def template_delta(pname, context):
+def template_delta(context):
```

Comment:
nice catch, so much for linting which should have seen this unused argument?

Reply:
it's entirely possible pylint does catch this, but doesn't consider it an error. I would have thought pyflakes could catch this though.